lib/reporters/beautify-stack.js

```diff
@@ -4,7 +4,7 @@ const stackUtils = new StackUtils({
 	ignoredPackages: [
 		'@ava/typescript',
 		'ava',
-		'nyc',
```

Comment:
We can keep `nyc` here for now. `c8` uses code coverage reports generated by Node.js and shouldn't appear in the stack traces.

Reply:
Sounds good to me. I've reverted this change so this file should have no changes now.

Reply:
Great!